lib/util.js

```diff
@@ -420,6 +420,78 @@ exports.nextSleepTime = function (
     Math.min(base, previousSleep * 3));
 };
 
+
+/**
+ * Return next sleep time calculated by the jitter rule.
+ *
+ * @param {Number} numofRetries 
+ * @param {Number} currentSleepTime 
+ * @param {Number} maxSleepTime 
+ * @returns {JSON} return next sleep Time and totalTime.
+ */
+exports.jitteredSleepTime = function (numofRetries, currentSleepTime, maxSleepTime, totalTimeout = 0, maxLoginTimeout = 300) {
+  const nextSleepTime = Math.min(maxSleepTime, 2 ** numofRetries);
+  let sleep = nextSleepTime + getJitter(currentSleepTime);
```

Comment:
Since nextSleepTime uses `2 ** numofRetries`, does that mean the first retry uses a base of 2 instead of 4?

Reply:
Changed the base to 4 and 'power' to 'multiply'